doc/nrf/known_issues.rst

```diff
@@ -1847,7 +1848,7 @@ KRKNWK-12482: Reception of correct frames can occasionally end in failure with e
 KRKNWK-11384: Assertion with Bluetooth® LE and multiprotocol usage
   The device might assert on rare occasions during the use of Bluetooth® LE and 802.15.4 multiprotocol.
 
```

Comment:
.. rst-class:: v2-1-0

KRKNWK-14898: CSMA-CA backoff parameters might not be randomized in an uniform way
  The backoff parameters of the CSMA-CA operation are generated by the logic which might not return the uniformly distributed random numbers.

Reply:
Added.